<!-- Launch file for localinos. Edit the default letter to correpsond to the localino -->

<launch>

	<arg name="name" default="case" />	 <!-- Name of the robot w/ the localino -->
	<arg name="localinos" default="BD" /> 	 <!-- letters of all localinos in experiment -->
	<arg name="num_meas" default="10" />	 <!-- # of packets to average out on each range measurment 
-->
	

<node name="localino_node" pkg="localino" type="localino.py" output="screen">
      <param name="localinos"	    value="$(arg localinos)" />
      <param name="num_meas"	    value="$(arg num_meas)" />
</node>

</launch>
